tests/stats_test.py

```diff
@@ -1,41 +1,34 @@
-from nuodb_test_class import *
+from .nuodb_test_class import *
 
 
 class TestNuoDB(NuoDBTelegrafTestClass):
     database_name = "nuodb"
```

Comment:
This is basically `bucket_name` or just `bucket` right?

Reply:
Yes. I will change the variable name.